report/report.go

```diff
@@ -177,6 +189,14 @@ func MakeReport() Report {
 			WithShape(Pentagon).
 			WithLabel("daemonset", "daemonsets"),
 
+		StatefulSet: MakeTopology().
+			WithShape(Triangle).
+			WithLabel("stateful set", "stateful sets"),
+
+		CronJob: MakeTopology().
+			WithShape(Triangle).
+			WithLabel("cron job", "cron jobs"),
+
```

Comment:
Let's go with
- Heptagon for Deployment - rationale: same shape as Service, which typically have a corresponding Deployment
- Triangle for CronJob - rationale: weird shape for weird thing
- Triangle for ReplicaSet - rationale: weird shape for thing we never show
- Pentagon for DaemonSet - rationale: daemons are evil
- Octagon for StatefulSet - rationale: last shape left